pubsub/oauth.go

```diff
@@ -100,25 +101,25 @@ func (a oauth) Login() (*LoginResponse, error) {
 		strings.NewReader(body.Encode()),
 	)
 	if err != nil {
-		return nil, fmt.Errorf("failed to make login req: %w", err)
+		return nil, errors.Errorf("failed to make login req: %s", err)
```

Comment:
might be missing something here, but why are we using `%s` instead of wrapping the error with `%w`?

Reply:
Errors doesnt support error-wrapping directive %w :( it will start complaining if we add %w

Reply:
This is exactly what `Errorf(..)` is supposed to do, you should be using `%w`. 

What error are you see if %w is used?

Reply:
Actually ignore this, was using wrong package for errors 🙈 I'll return wrapping the errors with %w